### ISSUE TITLE
Data management: Improve error handling when can't communicate with service(s)

### ISSUE THREAD

**Issue by alechenninger** [lightblue-platform/lightblue-applications#65]
Currently if you try to talk to a lightblue host that you can't, say you don't have a cert that you need, you'll get this in your browser's console:

> Failed to load resource: net::ERR_BAD_SSL_CLIENT_AUTH_CERT

This is ALL you get though, and the UI does nothing to tell you about this. So the UI could be nice and inform you about these kinds of errors.

**Comment by derek63** [comment]
It would also be nice to have a friendly error message for the next error we encountered if you have a valid cert, but the server is misconfigured for CORS support:

[Error] XMLHttpRequest cannot load https://lightbluedevserver.example.com/rest/metadata. Origin https://lightbluedevserver.example.com is not allowed by Access-Control-Allow-Origin. (data, line 0)

**Comment by alechenninger** [comment]
Waiting on #89 to use angular-ui-bootstrap alert for these
